Dockerfile

```diff
@@ -5,16 +5,18 @@ ENV LANG C.UTF-8
 WORKDIR /srv
 
 # System dependencies
-RUN apt-get update && apt-get install --yes python3-pip net-tools
-
-# Set git commit ID
-ARG COMMIT_ID
```

Comment:
We do still need to pass a revision ID through to the image. Although the more modern way to do this is to use `TALISKER_REVISION_ID`, which avoids us having to write it to a file. E.g.:

https://github.com/canonical-web-and-design/docs.snapcraft.io/blob/b3e01feba8446ce74c9384bae416ffd9a6ce10bd/Dockerfile#L15-L17